.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint Code
+
+on:
+  pull_request:
+    branches: [ develop ]
```

Comment:
You should make it run automatically on any PR to be able to fix the linting before merging.